src/test/resources/feedback/workermessage1.txt

```diff
@@ -10,15 +10,12 @@ Rating:
 Feedback: Foobar1
 Quality: 1
 Rating: 1
-
 Rating:
 
 Feedback: Foobar2
 Quality: 2
 Rating: 2
-
 Additional feedback: awes0me Message
-
```

Comment:
I'd say, functionality should be same or better after changes :stuck_out_tongue_winking_eye: Especially when i don't see, why this change was needed :wink: